stack-citationapi/recipes/deploy-citation-formatting-api.rb

```diff
@@ -0,0 +1,30 @@
+include_recipe 'stack-citationapi::role-phpapp'
+include_recipe 'redis::default'
+include_recipe 'stack-citationapi::deploy-publicapi'
```

Comment:
The lines above should be in the role, not in deploy (and last one has still the old name)